_pages/en_US/dumping-nand.md

```diff
@@ -5,26 +5,32 @@ redirect_from:
   - /dump-nand
 ---
 
-In this step, we'll be making a NAND backup. This is a 1:1 copy of your DSi internal storage, which we can use later on to setup hiyaCFW and NO$GBA.
+In this step, we'll be making a NAND backup. This is a 1:1 copy of your Nintendo DSi internal storage, which we can use later on to setup hiyaCFW and NO$GBA.
 
-We highly recommend that you do this. A NAND backup can be used as a restore point in the future, in case your DSi stops working.
+We highly recommend that you do this. A NAND backup can be used as a restore point in the future, in case your Nintendo DSi stops working.
 {: .notice--primary}
 
-## Instructions
+## Section I - SD card setup
+Using a Windows, Linux or Mac OS device? We have a tool that will automatically set up your SD card for following the guide.
```

Comment:
`macOS` for proper stylization points, Mac OS was back in the 90's ;P
(I don't really care that much, but it does look a bit odd seeing it written like that)